assembl/fabfile.py

```diff
@@ -1031,15 +1040,12 @@ def database_dump_aws():
 
 
 _processes_to_restart_without_backup = [
-    "dev:pserve", "celery", "changes_router",
-    "celery_notify", "source_reader"]
+    "dev:pserve" "celery", "changes_router",
```

Comment:
Missing a comma between "dev:pserve" and "celery" ?

Reply:
Already resolved. You were likely looking commit by commit?